pilot/cmd/pilot-agent/main.go

```diff
@@ -141,6 +143,19 @@ var (
 			role.TrustDomain = spiffe.GetTrustDomain()
 			log.Infof("Proxy role: %#v", role)
 
+			// Watch default cert locations if none were set
+			if len(tlsCertsToWatch) == 0 {
+				tlsCertsToWatch = []string{
+					path.Join(model.AuthCertsPath, model.CertChainFilename),
+					path.Join(model.AuthCertsPath, model.KeyFilename),
+					path.Join(model.AuthCertsPath, model.RootCertFilename),
+				}
+				if role.Type == model.Ingress {
```

Comment:
I think yu can get rid of this ingress cert thing.. and restrict this to just the mTLS certs. we are using SDS for ingress and gateways. and the certs specified in both do not have to have a standard path. So thsi code is likely a legacy leftover..